### DISCUSSION TITLE
macos ventura - new install machine possible?

### DISCUSSION THREAD

**Discussion by Techinbed** [daliansky/XiaoMi-Pro#710] [Q&A]
Hello
It's been a long time since I've been here and today I received back my old Xiomai Notebook pro from my kid and I want to delete the windows and install Mac OS.

Is there a way to install the latest macOS version ( venture) With the installation manual?

**Comment by stevezz3** [comment]
Hi, happy to welcome you back! The installation instructions have no changes since High Sierra. Just use the latest EFI build and everything will be fine.
